test/e2e/upgrade/alert/alert.go

```diff
@@ -108,7 +108,12 @@ func (t *UpgradeTest) Test(f *framework.Framework, done <-chan struct{}, upgrade
 			},
 		},
 	}
-	allowedFiringAlerts := helper.MetricConditions{}
+	allowedFiringAlerts := helper.MetricConditions{
+		{
+			Selector: map[string]string{"alertname": "TargetDown", "namespace": "openshift-e2e-loki"},
+			Text:     "Loki is nice to have, but we can allow it to be down briefly",
```

Comment:
`TargetDown` is already [`for: 15m`][1], so not clear to me that we can claim "briefly" when allowing it to fire.  Dropping "briefly" and just saying `...but we can allow it to be down.` would be internally consistent.

[1]: https://github.com/openshift/cluster-monitoring-operator/blob/4c19e806220b7b2db11449c52fc024811a25f489/assets/cluster-monitoring-operator/prometheus-rule.yaml#L28